### DISCUSSION TITLE
开启提示错误并且无法使用

### DISCUSSION THREAD

**Discussion by wayne880507** [qier222/YesPlayMusic#1498] [Q&A]
![螢幕擷取畫面 2022-04-05 020014](https://user-images.githubusercontent.com/94697642/161594567-3a68c0cc-be0f-4dd7-8148-0027b3aee061.png)
系统是windows11版本是最新版安装板跟免安装版都一样的问题

点选确定后卡在这里
![螢幕擷取畫面 2022-04-05 020107](https://user-images.githubusercontent.com/94697642/161594936-0e076340-f279-4104-ae7d-f772a2d881b7.png)

**Comment by qier222** [comment]
重启电脑试试

**Comment by pan93412** [comment]
https://github.com/qier222/YesPlayMusic/wiki/YesPlayMusic-使用中可能遇到的问题-(FAQ)#应用显示-error-listen-eacces-permission-denied-000010754
